package.json

```diff
@@ -19,7 +19,7 @@
     "start:tokens": "yarn workspace @twilio-paste/design-tokens build:dev",
     "start:website": "yarn workspace @twilio-paste/website develop",
     "watch:icons": "yarn workspace @twilio-paste/icons watch",
-    "prebuild": "node ./tools/build/pre-build.js",
+    "prebuild": "yarn bootstrap && node ./tools/build/pre-build.js",
```

Comment:
The action just run `yarn install` but we need to bootstrap. It recommends a postinstall or prebuild hook to run any install commands you need for the build

Reply:
I don't think we should add the bootstrap step in here because it will cause double runs with clean.  See my other comment